### DISCUSSION TITLE
issue: Chakra Ui Checkbox won't change visually (rerender) with setValue but the useForm value does

### DISCUSSION THREAD

**Discussion by AlternativeForce** [react-hook-form/react-hook-form#8318] [Q&A] [answered]
Since, i'm not able to share the code I have I just decided to fork a chakra ui and material ui example and change them to just show the differences.

For some reason the one from material ui changes but the chakra ui one doesn't? I'm relatively new at reactjs, been at it for 6~8 months, so still a novice. But what's the issue? The project I'm working on uses chakra ui, and I don't feel like switching just because of this,

I'm not sure if this is just chakra ui, react hook form, or both that are contributing to this.

material ui: https://codesandbox.io/s/react-hook-form-basic-forked-2kdwq3
chakra ui: https://codesandbox.io/s/chakra-ui-forked-3gr4on

Help is much appreciated!

**Comment by AlternativeForce** [comment, marked as answer]
Well turns out I was an idiot, answer was in front of me the whole time, checked is not a prop is ChakraUI, it is "isChecked". No need for an answer, but will say that react hook form has been really nice to me, thanks very much for it.

**Reply by bluebill1049** [reply]
awesome to hear that you sort it out.

**Reply by vv15software** [reply]
can you elaborate on that? so one way to change the value is via setValue react form hook. do we need to change isChecked in addition to setValue?
